configs/grid_rcnn/grid_rcnn_r50_fpn_gn-head_1x_coco.py

```diff
@@ -0,0 +1,11 @@
+_base_ = ['../grid_rcnn/grid_rcnn_r50_fpn_gn-head_2x_coco.py']
+# learning policy
+lr_config = dict(
+    policy='step',
+    warmup='linear',
+    warmup_iters=500,
+    warmup_ratio=1.0 / 3,
```

Comment:
0.001 is more commonly used in MMDet V2.0. The difference is marginal but it can stabilize training.

Reply:
done. thanks :smile: